airflow/sensors/sql_sensor.py

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from typing import Iterable
+from typing import Iterable, Callable
```

Comment:
I think you may be able to remove the `Iterable` import.

Reply:
I'd like to but the template fields on line 54 use the type annotation, thats probably taking down the Python 2 tests, should I clear those out as well (thats not my modification)?

Reply:
Ah sorry; I missed that. 😅